package.json

```diff
@@ -30,6 +31,7 @@
     "chai": "^4.3.6",
     "config": "^3.3.7",
     "cors": "^2.8.5",
+    "crypto-js": "^4.1.1",
```

Comment:
Since we are in nodejs env, we do have access to native crypto module.
We can use hash function from crypto module.
Eg:

const myhash = crypto
        .createHash("sha256")
        .update("THIS WILL BE HASHED")
        .digest("hex");